cylc/flow/suite_files.py

```diff
@@ -595,20 +644,70 @@ def register(reg=None, source=None, redirect=False, rundir=None):
 
 
 def create_auth_files(reg):
-    """Create or renew passphrase and SSL files for suite 'reg'."""
-    # Suite service directory.
-    srv_d = get_suite_srv_dir(reg)
-    os.makedirs(srv_d, exist_ok=True)
-
-    # If necessary, generate directory holding (in separate sub-dirs) the
-    # server (suite) public and private keys for authentication:
-    ensure_suite_keys_exist(srv_d)
-
-    # Create a new passphrase for the suite if necessary.
-    if not _locate_item(SuiteFiles.Service.PASSPHRASE, srv_d):
-        import random
-        _dump_item(srv_d, SuiteFiles.Service.PASSPHRASE, ''.join(
-            random.sample(PASSPHRASE_CHARSET, PASSPHRASE_LEN)))
+    """Create or renew authentication keys for suite 'reg' in the .service
+     directory."""
+
+    suite_srv_dir = get_suite_srv_dir(reg)
+
+    keys = {"client_public_key": KeyInfo(
+            KeyType.PUBLIC,
+            KeyOwner.CLIENT,
+            suite_srv_dir=suite_srv_dir),
+            "client_private_key": KeyInfo(
+            KeyType.PRIVATE,
+            KeyOwner.CLIENT,
+            suite_srv_dir=suite_srv_dir),
+            "server_public_key": KeyInfo(
+            KeyType.PUBLIC,
+            KeyOwner.SERVER,
+            suite_srv_dir=suite_srv_dir),
+            "server_private_key": KeyInfo(
+            KeyType.PRIVATE,
+            KeyOwner.SERVER,
+            suite_srv_dir=suite_srv_dir)
+            }
+
+    # WARNING, DESTRUCTIVE. Removes old key folders if they already exist.
+    for k in keys.values():
+        if os.path.exists(k.key_path):
+            shutil.rmtree(k.key_path)
+        os.makedirs(k.key_path, exist_ok=True)
+        os.chmod(k.key_path, stat.S_IRUSR | stat.S_IWUSR | stat.S_IXUSR)
```

Comment:
I don't think we need execute permissions.

```suggestion
        os.chmod(k.key_path, stat.S_IRUSR | stat.S_IWUSR)
```